### PR TITLE
Cell Table Assumptions

### LINKED ISSUE #789: Describe assumptions/structure of cell table
**Is your feature request related to a problem? Please describe.**
The current pipeline assumes that the cell table has a specific format. In particular, there are specific pre-channel and post-channel columns that we use to figure out where the image data is. This is not documented anywhere, and in general the format of the cell table is not explicitly described. See e.g. https://github.com/angelolab/ark-analysis/issues/787#issuecomment-1290034760

**Describe the solution you'd like**
We should add a section describing what the columns of the cell table mean, which are required (if generated by a different pipeline, for example), and how to change the defaults in settings.py. Some of this information is here: https://ark-analysis.readthedocs.io/en/latest/_rtd/data_types.html. However, this is not prominently linked throughout ark. We should either flesh this out with the required info and add links, or move this info into the readme.

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #789.

**How did you implement your changes**

Links the `data_types.md` document in the README. Added an example column header for the cell table. Added instructions about adjusting column header values for the cell table if necessary.

**Remaining issues**

- [ ] Might be missing a few things.

### CODE REVIEW

File: docs/_rtd/data_types.md [hunk 1]
```diff
@@ -23,27 +23,55 @@ Shape: 2D matrix of cells x cell_features
...
-* `area`: area of the cell
-* `eccentricity`: eccentricity of the ellipse that has the same second-moments as the region. The eccentricity is the ratio of the focal distance (distance between focal points) over the major axis length. The value is in the interval [0, 1). When it is 0, the ellipse becomes a circle.
-* `major_axis_length`: the length of the major axis of the ellipse that has the same normalized second central moments as the region
-* `minor_axis_length`: the length of the minor axis of the ellipse that has the same normalized second central moments as the region.
-* `perimeter`: perimeter of object which approximates the contour as a line through the centers of border pixels using a 4-connectivity
-* `convex_area`: the area of the convex hull
-* `equivalent_diameter`: the diameter of the circle with the same area as the cell
-* `centroid-0`: the x-coordinate of the centroid
-* `centroid-1`: the y-coordinate of the centroid
+* `channel_1` -> `channel_n`: The unique channels for each marker used.
```
Comment (ngreenwald) [inline docs/_rtd/data_types.md:34]:
```suggestion
* `channel_1` -> `channel_n`: The amount of signal from that channel in the given cell. The way this is calculated depends on how the cell table was generated. The default is that this is the size-normalized counts, but there is also the option for this to be arcsin-h transformed. 
```

File: docs/_rtd/data_types.md [hunk 2]
```diff
@@ -23,27 +23,55 @@ Shape: 2D matrix of cells x cell_features
...
+* `major_minor_axis_ratio`: the major axis length divided by the minor axis length. 
+* `perim_square_over_area`: the square of the perimeter divided by the area. 
+* `major_axis_equiv_diam_ratio`: the major axis length divided by the equivalent diameter. 
+* `convex_hull_resid`: the difference between the convex area and the area divided by the convex area. 
+* `centroid_dif`: the normalized euclidian distance between the cell centroid and the corresponding convex hull centroid. 
+* `num_concavities`: the number of concavities of the region 
+* `nc_ratio`: for nuclear segmentation only. The nuclear area divided by the total area. 
+
+If you are using your own spatial analysis tools to generate information for the Cell Table, adjust the following values in `ark/.settings.py`:
+* `PRE_CHANNEL_COL`: From `CELL_SIZE` to the parameter which occurs before the channel names.
```
Comment (ngreenwald) [inline docs/_rtd/data_types.md:57]: This wording makes it sound like they should specify the range of values between cell_size and their value. I would update it to just tell that what value to place here

File: docs/_rtd/data_types.md [hunk 3]
```diff
@@ -23,27 +23,55 @@ Shape: 2D matrix of cells x cell_features
...
+* `POST_CHANNEL_COL`: From `CELL_LABEL` to the parameter which occurs after the last channel name.
+
+Afterwards the Docker Image can be started with 
+```sh
+start_docker.sh
+```
+
+It's recommended to adjust these values before starting the docker image, however if necessary you may do so within the docker image itself. First open a terminal within the Jupyter Lab UI and run the following commands:
+```sh
```
Comment (ngreenwald) [inline docs/_rtd/data_types.md:66]: I think this got copied from somewhere else

**Review by ngreenwald — CHANGES_REQUESTED** [review]
A couple changes

File: docs/_rtd/data_types.md (hunk 3 above)
Comment (srivarra) [reply docs/_rtd/data_types.md:66]: It was from [here](https://github.com/angelolab/ark-analysis/issues/787#issuecomment-1289781226).

**Review by ngreenwald — APPROVED** [review]
Looks good

**Pull request merged**